src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1713,18 +1727,26 @@ internal sealed class LegacySdcaBinaryTrainer : SdcaBinaryTrainerBase<IPredictor
         /// </summary>
         public sealed class Options : BinaryOptionsBase
         {
-            [Argument(ArgumentType.Multiple, HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
-            public ISupportSdcaClassificationLossFactory LossFunction = new LogLossFactory();
+            [Argument(ArgumentType.Multiple, Name = "LossFunction", HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
+            public ISupportSdcaClassificationLossFactory LossFunctionFactory = new LogLossFactory();
+
+            public ISupportSdcaClassificationLoss LossFunction;
```

Comment:
Please add user-facing document.

Reply:
changes are removed for this class

---
In reply to: [262724466](https://github.com/dotnet/machinelearning/pull/2856#discussion_r262724466) [](ancestors = 262724466)